unittests/test_clair/vulnerabilities-check_test.rego

```diff
@@ -0,0 +1,21 @@
+package required_checks
+
+test_violation_critical_vulnerabilities {
+    result := violation_critical_vulnerabilities with input as data.clair
+    contains(json.marshal(result), "clair_critical_vulnerabilities")
```

Comment:
I would like to improve the performance with a better statement like <expression_a> == "clair_critical_vulnerabilities", but I have not figured out how to get it from `result`.

Reply:
I have not find a solutions. Welcome to share your ideas. Thank you!

Reply:
This might work:
`result[_].details.name == "clair_critical_vulnerabilities"`

You could also add more checks for other parts of the result this way, like the `msg` field if it's relevant to the check.

Reply:
@dirgim Thank you so much!! It works. I add one more check for the msg to unittests.